setup.py

```diff
@@ -110,6 +110,27 @@
   DEFINE_MACROS += (('PyMODINIT_FUNC', pymodinit),)
 
 
+# By default, Python3 distutils enforces compatibility of
+# plugins with the OSX version Python3 was built with.
+# For Python3.4, this is OSX 10.6, but we need Thread Local Support (__thread)
+# from OSX 10.7
+def get_os_version_osx():
+  import sysconfig, re
+  platform = sysconfig.get_platform()
+  platform_match = re.match("macosx-(.*)-.*", platform)
+  if platform_match:
+    if platform_match.groups()[0] <= "10.6":
```

Comment:
I think we want to use ```distutils.version.StrictVersion``` for comparison rather than string comparison, otherwise we have things like ```"9.4" > "10.6"```